docs/markdown/snippets/pgi_compilers.md

```diff
@@ -0,0 +1 @@
+added PGI C, C++ and Fortran compiler support
```

Comment:
This does not follow the specified format for snippets. There should be a headline with two hash marks followed by body text. Something like this:

```
## Added PGI compiler support

PGI's C, C++ and Fortran compilers are now supported [add potential limitations here, such as if they have only been tested on platform X and so on]
```

Reply:
done